arch/riscv/mach-spike/Kconfig

```diff
@@ -150,6 +150,19 @@ endmenu
 
 menu "Shutdown scheme"
 
+config SPIKE_SHUTDOWN_SPIKE
+	bool "Spike HTIF poweroff"
+	help
```

Comment:
And adds
select SBI_CSR_MIMPID if SBI
here.

Reply:
done